.gitmodules

```diff
@@ -1,5 +1,5 @@
 [submodule "dotCMS/src/main/enterprise"]
 	path = dotCMS/src/main/enterprise
 	url = git@github.com:dotCMS/enterprise.git
-	branch = master
+	branch = issue-19234-hookup-metadata-api
```

Comment:
please rollback this change

Reply:
Once I'm done with this whole thing and we know we're ready to merge I will